artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/tools/xml/XmlDataImporter.java

```diff
@@ -90,10 +91,12 @@ public final class XmlDataImporter extends ActionAbstract {
    private ClientSession session;
    private ClientProducer producer;
 
-   @Option(names = "--host", description = "The host used to import the data. Default: localhost.")
-   public String host = "localhost";
+   @Deprecated(forRemoval = true)
+   @Option(names = "--host", description = "The host used to import the data. Default: null. Deprecated! Use 'url' instead.")
+   public String host = null;
 
-   @Option(names = "--port", description = "The port used to import the data. Default: 61616.")
+   @Deprecated(forRemoval = true)
+   @Option(names = "--port", description = "The port used to import the data. Default: 61616. Deprecated! Use 'url' instead.")
```

Comment:
just use hidden

Reply:
eg: https://github.com/apache/activemq-artemis/blob/7742936583d48e179f67c7d73b7b49cbe83731d5/artemis-cli/src/main/java/org/apache/activemq/artemis/cli/commands/messages/DestAbstract.java#L41-L42


if you have a script showing the older parameters, it should work fine, but it will prevent users from discovering the deprecated term.


it is the best approach on the CLI.. hidden=true